Api/V8/JsonApi/Response/ErrorResponse.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace Api\V8\JsonApi\Response;
+
+class ErrorResponse implements \JsonSerializable
+{
+    /**
+     * @var integer
+     */
+    private $status;
+
+    /**
+     * @var string
+     */
+    private $title;
+
+    /**
+     * @var string
+     */
+    private $detail;
+
+    /**
+     * @return integer
+     */
+    public function getStatus()
+    {
+        return $this->status;
+    }
+
+    /**
+     * @param integer $status
+     */
+    public function setStatus($status)
+    {
+        $this->status = $status;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    /**
+     * @param string $title
+     */
+    public function setTitle($title)
+    {
+        $this->title = $title;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDetail()
+    {
+        return $this->detail;
+    }
+
+    /**
+     * @param string $detail
+     */
+    public function setDetail($detail)
+    {
+        $this->detail = $detail;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function jsonSerialize()
+    {
+        $response = [
+            'errors' => [
+                'status' => $this->getStatus(),
+                'title' => $this->getTitle(),
+                'detail' => $this->getDetail(),
+            ]
+        ];
+
+        return array_filter($response);
```

Comment:
array_filter can not change the input array whit this array structure. did you mean array_filter($response['errors']) first?

Reply:
Hehe, yeah, my mistake. Was a copy paste. We don;t use filter here.

Reply:
Fixed.